set.mm

```diff
@@ -160919,6 +160936,38 @@ reflection about the origin (under the map ` x |-> -u x ` ).  (Contributed
     cc adantr absid sylan eqtr3d ex sylbid imp ) ABCZADEFZAGHZAIZJZUEUFDUHEFZUI
     AKUEUJUIUEUJLUHGHZUGUHUEUKUGJZUJUEAQCULAMANORUEUHBCUJUKUHJAPUHSTUAUBUCUD $.
 
+  $( Alternate expression for the absolute value of a real number.
+     (Contributed by RP, 11-May-2024.) $)
+  reabsifneg $p |- ( A e. RR -> ( abs ` A ) = if ( A < 0 , -u A , A ) ) $=
+    ( cr wcel cc0 clt wbr cneg cif cabs cfv wa cle wceq wi ltle mpan2 imdistani
+    0re absnid eqcomd syl wn 0red id lenltd bicomd pm5.32i absid sylbi ifeqda )
+    ABCZADEFZAGZAHAIJZUKULUMAUNUKULKZUNUMUOUKADLFZKUNUMMUKULUPUKDBCULUPNRADOPQA
+    SUATUKULUBZKZUNAURUKDALFZKUNAMUKUQUSUKUSUQUKDAUKUCUKUDUEUFUGAUHUITUJT $.
+
+  $( Alternate expression for the absolute value of a real number.
+     (Contributed by RP, 11-May-2024.) $)
+  reabsifnpos $p |- ( A e. RR -> ( abs ` A ) = if ( A <_ 0 , -u A , A ) ) $=
+    ( cr wcel cc0 cle wbr cneg cif cabs cfv wa absnid eqcomd wn wceq wi 0re clt
+    ltnle ltle sylbird mpan imdistani absid syl ifeqda ) ABCZADEFZAGZAHAIJZUGUH
+    UIAUJUGUHKUJUIALMUGUHNZKZUJAULUGDAEFZKUJAOUGUKUMDBCZUGUKUMPQUNUGKUKDARFUMDA
+    SDATUAUBUCAUDUEMUFM $.
+
+  $( Alternate expression for the absolute value of a real number.
+     (Contributed by RP, 11-May-2024.) $)
+  reabsifpos $p |- ( A e. RR -> ( abs ` A ) = if ( 0 < A , A , -u A ) ) $=
+    ( cr wcel cc0 clt wbr cneg cif cabs cfv wa cle wceq 0re ltle mpan imdistani
+    wi absid eqcomd syl wn id 0red lenltd pm5.32i absnid sylbir ifeqda ) ABCZDA
+    EFZAAGZHAIJZUJUKAULUMUJUKKZUMAUNUJDALFZKUMAMUJUKUODBCUJUKUORNDAOPQASUATUJUK
+    UBZKZUMULUQUJADLFZKUMULMUJURUPUJADUJUCUJUDUEUFAUGUHTUIT $.
+
+  $( Alternate expression for the absolute value of a real number.
+     (Contributed by RP, 11-May-2024.) $)
+  reabsifnneg $p |- ( A e. RR -> ( abs ` A ) = if ( 0 <_ A , A , -u A ) ) $=
+    ( cr wcel cc0 cle wbr cneg cif cabs cfv wa absid eqcomd wn wi 0re clt ltnle
+    wceq ltle sylbird mpan2 imdistani absnid syl ifeqda ) ABCZDAEFZAAGZHAIJZUGU
+    HAUIUJUGUHKUJAALMUGUHNZKZUJUIULUGADEFZKUJUISUGUKUMUGDBCZUKUMOPUGUNKUKADQFUM
+    ADRADTUAUBUCAUDUEMUFM $.
+
```

Comment:
Do we really need all of these variants (in main)? As far as I can see, only ~reabsifneg is actually used in the following.

Reply:
Because I needed to map numbers <0, =0, >0 to -1, 1, and 1 (and not -1, 0, and 1 as with df-sgn), I needed one of these. The others were for future authors who might have made different choices. But it looks like all of this is going to my sandbox.

Reply:
Now that they are in my sandbox, a fifth theorem based on `df-abs` was added.